tests/phpunit/CRM/Contribute/BAO/ContributionTest.php

```diff
@@ -253,8 +246,8 @@ public function testcreateAndGetHonorContact() {
     $this->assertEquals(array(
       $id => array(
         'honor_type' => 'In Honor of',
-        'honorId' => $id,
-        'display_name' => 'John Doe',
+        'honorId' => $contactId,
```

Comment:
The $id value turned out to be wrong - but as the DB was empty both contribution & contact had the same id (1). Adding the tear down function meant the default DB had 2 * domain contacts & this 'pass by fluke' was no longer solid enough.